<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVCameraCalibrationEmbeddedWidget</class>
 <widget class="QWidget" name="CVCameraCalibrationEmbeddedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="mpBackwardButton">
       <property name="minimumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 75px;
    min-height: 36px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QPushButton:checked {
    background-color: green
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../CVDevLibrary/resources/resources.qrc">
         <normaloff>:/icons/tango/32x32/go-previous.png</normaloff>:/icons/tango/32x32/go-previous.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="mpForwardButton">
       <property name="minimumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 75px;
    min-height: 36px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QPushButton:checked {
    background-color: green
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../CVDevLibrary/resources/resources.qrc">
         <normaloff>:/icons/tango/32x32/go-next.png</normaloff>:/icons/tango/32x32/go-next.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="mpUndistCheckBox">
       <property name="text">
        <string>Undist</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="mpNoImageLCD">
       <property name="minimumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="digitCount">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="mpAutoCapCheckBox">
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Auto</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="mpCaptureButton">
       <property name="minimumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 75px;
    min-height: 36px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QPushButton:checked {
    background-color: green
}</string>
       </property>
       <property name="text">
        <string>Capture</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="mpCalibrateButton">
       <property name="minimumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 75px;
    min-height: 36px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QPushButton:checked {
    background-color: green
}</string>
       </property>
       <property name="text">
        <string>Calibrate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="mpRemoveButton">
       <property name="minimumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 75px;
    min-height: 36px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QPushButton:checked {
    background-color: green
}</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>7</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string> Total Images</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="mpTotalImageLCD">
       <property name="minimumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="digitCount">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpExportButton">
       <property name="minimumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 75px;
    min-height: 36px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QPushButton:checked {
    background-color: green
}</string>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../CVDevLibrary/resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
